app/main/views/frameworks.py

```diff
@@ -1,26 +1,29 @@
+import datetime
+
 from flask import jsonify, abort, request
 from sqlalchemy.exc import IntegrityError, DataError
 from sqlalchemy import func, orm, case
-import datetime
-
 from dmapiclient.audit import AuditTypes
 from dmutils.config import convert_to_boolean
+
 from .. import main
 from ...models import db, Framework, DraftService, User, Supplier, SupplierFramework, AuditEvent, Lot
 from ...utils import (
-    get_json_from_request, json_has_required_keys, json_only_has_required_keys,
+    get_json_from_request,
+    json_has_required_keys,
+    json_only_has_required_keys,
+    list_result_response,
+    single_result_response,
     validate_and_return_updater_request,
 )
 from ...framework_utils import validate_framework_agreement_details_data
 
 
 @main.route('/frameworks', methods=['GET'])
 def list_frameworks():
-    frameworks = Framework.query.all()
+    frameworks = Framework.query
 
-    return jsonify(
-        frameworks=[f.serialize() for f in frameworks]
-    )
+    return list_result_response("frameworks", frameworks)
```

Comment:
Probably don't need a separate variable at this point, but :man_shrugging:

Reply:
Will tidy